service/src/main/resources/api/service_openapi.yaml

```diff
@@ -152,6 +152,34 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/profiles/v1/{profileId}/spendReport:
```

Comment:
Some alternatives:

```
/api/azure/v1/spendReport/{profileId}
/api/spendReporting/v1/azure/{profileId}
```

I'm not too worried about the specifics of this right now. The most important thing is that the API will accept a billing profile id and return a spend report for that profile as long as it is an Azure profile.